### PR TITLE
Update translation strings, update German, more formspec escaping

### PR DESCRIPTION
**Pull request by Wuzzy2**
This PR does the following:

* Update outdated strings and put them into `template.txt`
* Update/improve German translation
* Make sure all translated strings are formspec-escaped before they go into formspecs

I hope I didn't break anything. ;-)

### CODE REVIEW

File: api.lua [hunk 1]
```diff
@@ -158,7 +159,7 @@ end
 
 
 unified_inventory.register_craft_type("normal", {
-	description = "Crafting",
+	description = F("Crafting"),
```
Comment (HybridDog) [inline api.lua]: @Wuzzy2 the brackets are not necessary and because the function is used for translation it can be considered as special to ratify the different code style:

+	description = F"Crafting",

Comment (Wuzzy2) [inline api.lua]: So what?
`S("blabla")` exists throughout the code (with brackets).

I won't change coding style without VannessaE wanting it.

Comment (VanessaE) [inline api.lua]: I am not adverse to changes in code style, so long as you stick to tabs like G*d intended. :smile:   It would be a good idea to keep such changes to separate commits though.

(I am, however, adverse to my name being misspelled :stuck_out_tongue:)

Comment (HybridDog) [inline api.lua]: You're right, it should be done in separate commit(s) (maybe in another PR).
